src/main/resources/static/domain-events-api.yml

```diff
@@ -1127,6 +1163,66 @@ components:
         - reason
         - legacyReasonCode
         - destination
+    ApplicationAppealMadeEnvelope:
+      type: object
+      properties:
+        id:
+          $ref: '#/components/schemas/EventId'
+        timestamp:
+          type: string
+          example: '2022-11-30T14:53:44'
+          format: date-time
+        eventType:
+          type: string
+          example: approved-premises.application.appeal-made
+        eventDetails:
+          $ref: '#/components/schemas/ApplicationAppealMade'
+      required:
+        - id
+        - timestamp
+        - eventType
+        - eventDetails
+    ApplicationAppealMade:
+      type: object
+      properties:
+        applicationId:
+          $ref: '#/components/schemas/ApplicationId'
+        applicationUrl:
+          $ref: '#/components/schemas/ApplicationUrl'
+        appealId:
+          $ref: '#/components/schemas/AppealId'
+        appealUrl:
+          $ref: '#/components/schemas/AppealUrl'
+        personReference:
+          $ref: '#/components/schemas/PersonReference'
+        deliusEventNumber:
+          $ref: '#/components/schemas/DeliusEventNumber'
+        createdAt:
+          type: string
+          example: '2022-11-30T14:51:30'
+          format: date-time
+        appealDetail:
+          type: string
+        decision:
+          $ref: '#/components/schemas/AppealDecision'
+        decisionDetail:
+          type: string
```

Comment:
in addition, we're going to need to record:

- the user recording the appeal  decision, eg. `decisionRecordedBy` (`AppealEntity.createdBy()` I think)
- the person who made the decision (as reported by the `decisionRecordedBy` user), `AppealEntity.reviewer()` I think

Reply:
These fields have been added as `createdBy` and `reviewer`.